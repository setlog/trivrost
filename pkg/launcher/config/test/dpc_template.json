{
    "LauncherUpdate": [
        {
            "BundleInfoURL": "https://example.com/windows/launcher/bundleinfo.json",
            "TargetPlatforms": [ "{{.OS}}-{{.Arch}}" ]
        }
    ]
}
