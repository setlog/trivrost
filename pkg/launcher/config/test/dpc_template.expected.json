{
    "LauncherUpdate": [
        {
            "BundleInfoURL": "https://example.com/windows/launcher/bundleinfo.json",
            "TargetPlatforms": [ "AmigaOS-68000x16" ]
        }
    ]
}
